README.md

```diff
@@ -1,134 +1,174 @@
-  * ❌ Re-engaging of <ins>**LONGITUDINAL CONTROL**</ins> (acceleration/brake) on brake pedal release ❌ (was never included in MADS)
-  * ❌ No disengaging on ACC MAIN in OFF state ❌
+The following changes are a **VIOLATION** of this policy and **ARE NOT** included in any sunnypilot branches:
+* Driver Monitoring:
+  * ❌ "Nerfing" or reducing monitoring parameters.
+* Panda safety:
+  * ❌ No preventing disengaging of <ins>**LONGITUDINAL CONTROL**</ins> (acceleration/brake) on brake pedal press.
+  * ❌ No auto re-engaging of <ins>**LONGITUDINAL CONTROL**</ins> (acceleration/brake) on brake pedal release.
+  * ❌ No disengaging on ACC MAIN in OFF state.
 
-⚒ Installation
----
+</details>
 
-### Comma URL (Comma 3 only quick setup)
 
-8.14 was last updated in July 2022, many changes have been made since then so test-c3 is recommended
-```
-1. Factory reset/uninstall previous software if you already installed a branch
-2. Select Custom Software
-3. Input the following URL: installer.comma.ai/sunnyhaibin/test-c3
-```
+<details><summary><h3>⚒ Installation</h3></summary>
 
-### Comma 2 / non-test fork for comma 3
-To install sunnypilot's fork, simply use the installer.comma.ai URL (thanks [Shane](https://github.com/sshane/openpilot-installer-generator)!) on the setup screen for "Custom Software" after you factory reset or uninstalled sunnypilot from a previous install:
+ <details><summary>URL (Easy)</summary> 
 
-<b>HTTPS://</b> prepend is required for comma 2, This is not required for comma 3!
+  comma three
+  ------
+  
+   Please refer to [Recommended Branches](#-recommended-branches) to find your preferred/supported branch, this guide will assume you want to install the latest release:
 
-```
-https://installer.comma.ai/sunnyhaibin/<insert_branch_name>
-```
-For example, if you would like to install the branch for HKG:
+   1. [Factory reset/uninstall](https://github.com/commaai/openpilot/wiki/FAQ#how-can-i-reset-the-device) previous software if you already installed another fork.
+   2. Select `Custom Software` when given the option upon reboot.
+   3. Input the following URL: ```bit.ly/sp-test-c3```[^4] (note: `https://` is not requirement on the comma three)
+   4. Complete the rest of the installation following the onscreen instructions.
+   5. Pop into the `#installation-help` channel on Discord for further assistance.
+
+  comma two
+  ------
+
+  Please refer to [Recommended Branches](#-recommended-branches) to find your preferred/supported branch, this will assume you want to install the latest release:
 
-* [`0.8.12-prod-full-hkg`](https://github.com/sunnyhaibin/openpilot/tree/0.8.12-prod-full-hkg):
+  1. Factory reset/uninstall previous software if you already installed another fork.
+  2. Select `Custom Software` when given the option.
+  3. Input the following URL: ```https://smiskol.com/fork/sunnyhaibin/0.8.12-4-prod```
```

Comment:
```suggestion
  3. Input the following URL: ```https://smiskol.com/fork/sunnyhaibin/0.8.12-prod-full-_____```
```
0.8.12 based branch names are brand-specific. Maybe we include a link or a screenshot of the available `0.8.12-prod-full-____` branches here?

Reply:
Good point, could do it as a footnote to the either #-recommended-branches or a link / search query for the available branches, https://github.com/sunnyhaibin/sunnypilot/branches/all?query=0.8.12-prod-full ??